.github/workflows/pull_requests.yml

```diff
@@ -10,55 +10,60 @@ jobs:
 
     if: github.event_name == 'pull_request'
     steps:
-    - uses: actions/checkout@master
-      with:
-        ref: ${{ github.ref }}
+      - uses: actions/checkout@master
+        with:
+          ref: ${{ github.ref }}
 
-    - run: ./scripts/ci/prepare_linux.sh
-    - run: go test -race -cover
-    - run: gofmt -w -s .
-    - run: ./scripts/fail_on_diff.sh
+      - run: ./scripts/ci/prepare_linux.sh
+      - run: go test -race -cover
+      - run: gofmt -w -s .
+      - run: ./scripts/fail_on_diff.sh
 
   dsq-tests-ubuntu:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@master
-      with:
-        ref: ${{ github.ref }}
+      - uses: actions/checkout@master
+        with:
+          ref: ${{ github.ref }}
 
-    - run: ./scripts/ci/prepare_linux.sh
-    - run: go build -trimpath -buildmode=pie -mod=readonly -modcacherw
-    - run: ./scripts/test.py
+      - run: ./scripts/ci/prepare_linux.sh
+      - run: go build -trimpath -buildmode=pie -mod=readonly -modcacherw
+      - run: ./scripts/test.py
 
   dsq-tests-windows:
     runs-on: windows-latest
 
     steps:
-    - uses: actions/checkout@master
-      with:
-        ref: ${{ github.ref }}
+      - uses: actions/checkout@master
+        with:
+          ref: ${{ github.ref }}
 
-    - run: ./scripts/ci/prepare_windows.ps1
-      shell: pwsh
-    - run: go build -trimpath -buildmode=pie -mod=readonly -modcacherw
-    - run: ./scripts/test.py
-      shell: bash
+      - run: ./scripts/ci/prepare_windows.ps1
+        shell: pwsh
+      - run: go build -trimpath -buildmode=pie -mod=readonly -modcacherw
+      # if you call os.TempDir() that path would be \blabla\Runner~1\blabla the ~ will break some apps.
```

Comment:
Actually you found a bug in DataStation here! Tildes should be valid anywhere it was my own bad resolution code that broke it. Revert this change and update go.mod to take in https://github.com/multiprocessio/datastation/pull/228 when I merge that.

Reply:
Okay

Reply:
That PR is merged now. Give it a shot! I'm going to the doctor so I can approve/merge in 3 hours or so

Reply:
it works!